dashboard/src/modules/components/AlertComponent/index.jsx

```diff
@@ -19,7 +19,7 @@ function AlertMessage({ message, link }) {
       title={[
         `${message}`,
         <a className="alertHelpText">
-          Log into your account
+          {link}
         </a>,
```

Comment:
Is this "anchor" supposed to have a hypertext reference?  (I.e., does this actually produce a clickable link?)  Or, is there nowhere for it to link _to_, yet?  (I know Varshini is working on this, but I don't know whether it is "in", yet.)

If there's nowhere for it to go, should we mark this with a `TODO` comment to make it more obvious that it's not finished?